doc/python_reference.rst

```diff
@@ -664,6 +666,22 @@ Functions:
    restrict_forward_to_label
    restrict_forward_to_stc
 
+.. currentmodule:: mne.transforms
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   SphericalHarmonicTPSWarp
```

Comment:
I would like to avoid acronyms in name.

How about:

SphereSplineWarp
SphericalSplineWarp

?